common/containers/Tabs/SendTransaction/index.tsx

```diff
@@ -17,6 +17,7 @@ import SubTabs, { Tab } from 'components/SubTabs';
 import { getNetworkConfig } from 'selectors/config';
 import { isNetworkUnit } from 'utils/network';
 import { RouteNotFound } from 'components/RouteNotFound';
+import { getNonceRequested, TGetNonceRequested } from 'actions/transaction';
```

Comment:
A much better place to put this would be in the `GasSlider` `componentDidMount` component lifecycle method if we're going to be using that for anything gas price / gas limit /nonce/ related

Reply:
Great idea, gasSlider can handle getting its own nonce.

We may need to make updates to the reducer not to clear nonce since it's no longer really part of the transaction state in this case.